### DISCUSSION TITLE
运行build_pag.sh报错

### DISCUSSION THREAD

**Discussion by jackTJT** [Tencent/libpag#1929] [General] [answered]
在linux部署libpag后，运行build_pag.sh报错
![H{TL@}WAGWXA5MTP~ZRD1VK](https://github.com/Tencent/libpag/assets/96832322/c7372b16-6761-4fa0-9928-adb2a5c40e67)

**Comment by domchen** [comment, marked as answer]
忘记了安装 depsync 依赖同步工具，执行一下 `npm install -g depsync` 如果还是有报错建议更新一下代码到main分支最新版本。
